app/models/determination.rb

```diff
@@ -25,6 +26,14 @@ def calculate_total
     self.total = self.fees + self.expenses
   end
 
+  def calculate_vat
```

Comment:
I suspect that if existing determinations have VAT included the total will
reflect that, but the VAT line in the breakdown will be blank.  So the two
totals will be the same.

I'll check this though...

On 14 December 2015 at 12:12, Joel Sugarman <notifications@github.com>
wrote:

> In app/models/determination.rb
> <https://github.com/ministryofjustice/advocate-defence-payments/pull/836#discussion_r47490248>
> :
>
> > @@ -25,6 +26,14 @@ def calculate_total
> >      self.total = self.fees + self.expenses
> >    end
> >
> > +  def calculate_vat
>
> hey @aliuk2012 <https://github.com/aliuk2012>, discussed this with
> @EdwardAndress <https://github.com/EdwardAndress> but just to show you a
> local manual test where i zeroed the vat_amount for a determination and
> then viewed it in show view - i.e. how it would be on gamma. This probably
> needs conditional logic to not confuse users (or ask Product Management if
> they will accept this)
> [image: screen shot 2015-12-14 at 12 08 47]
> <https://cloud.githubusercontent.com/assets/7016425/11780672/a5d97a30-a25b-11e5-96d3-da7fa1aad60b.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/advocate-defence-payments/pull/836/files#r47490248>
> .
>